src/forms/input/Input.tsx

```diff
@@ -50,43 +56,67 @@ export class Input extends React.PureComponent<InputProps> {
         return <label className="clr-control-label">{label}</label>;
     }
 
-    render() {
+    private buildInput(className: any, uid: string) {
         const {
-            className, //prettier
+            style,
             disabled,
-            helperText,
-            label,
             value,
             defaultValue,
-            isBoxed,
             placeholder,
             size,
             type,
             children,
             name,
             id,
+            required,
+            onBlur,
+        } = this.props;
+        return (
+            <React.Fragment>
+                <input
+                    type={type || "text"}
+                    name={name}
+                    id={id ? id : uid}
```

Comment:
you can replace it by >
```suggestion
                    id={id || uid}
```
to go through js conventions

Reply:
done